### PR TITLE
Eg 92 attacker defender title

### LINKED ISSUE #92: Make Pokemon Title to say 'Attacker/Defender'
**Is your feature request related to a problem? Please describe.**
The current title for the proponent and opponent pokemon's on the versus screen currently says "Your Pokemon" and "Enemy Pokemon". There needs to be a better descriptive title when two pokemon's are dueling each other.

**Describe the solution you'd like**
The solution would be to change "Your Pokemon" to "Attacker" and "Enemy Pokemon" to "Defender".

### PR DESCRIPTION
**Pull request by BlackCubes**
## Changes
1. Updated the titles for user-poke and enemy-poke in both `SinglePokemon.js` and `EnemyPokemon.js` to be better descriptive for the user.
2. Updated the stylings for the titles on mobile and desktop to look better for the user.

## Purpose
There should be different titles for the user's pokemon and the opponent in order to show that the user is attacking the opponent while the opponent is defending itself.

## Approach
Since the user does not know if they are attacking against the opponent, the titles for the user's pokemon was changed to "Attacker" while the opponent was changed to "Defender". Lastly, the stylings were updated to reflect on the decrease of title word length for both mobile and desktop.

Closes #92

### CODE REVIEW

**Review by connielion — APPROVED** [review]
Mmm. Nice styling. Keep up the good work.

**Review by Lap343 — APPROVED** [review]
Nice small change and looks good in app

**Review by Wolftownn — APPROVED** [review]
Nice job as always Elias!

File: pokemon-app/src/styles/EnemyPokemon-mobile.css [hunk 1]
```diff
@@ -1,78 +1,84 @@
...
-        font-size: 1.5vw;
-    }
-}
\ No newline at end of file
+@media screen and (max-width: 700px), (max-height: 820px) {
+  .enemy-pokemon__display-name {
+    font-size: 1.7vw;
+    width: 110%;
+    border: 4px solid black;
+  }
```
Comment (Gjhnsn) [inline pokemon-app/src/styles/EnemyPokemon-mobile.css:6]: mobile friendly👀

**Review by Gjhnsn — APPROVED** [review]
I know this is gonna look so cool, good job!

**Pull request merged**